libpod/container_darwin.go

```diff
@@ -0,0 +1,23 @@
+// +build darwin
+
+package libpod
+
+type containerStateDarwin struct {
+}
+
+// containerStateLinux is intentionally left as a blank stub
+type containerStateLinux struct {
+}
+
+// containerStateWindows is intentionally left as a blank stub
+type containerStateWindows struct {
+}
+
+// IsRootless is a stub function that always returns false on darwin
+func IsRootless() bool {
+	return false
+}
+
+// BecomeRootlessInUserNamespace is a stub function that always returns false on darwin
+func BecomeRootlessInUserNamespace() (bool, error) {
```

Comment:
Why is this in libpod and not the rootless package?

Reply:
moved.

Reply:
> moved

Did you forget to commit and/or push this move?  Your current tip still has the stubs in `libpod/container_darwin.go`:

```console
$ git fetch origin
$ git log --oneline -1 origin/pr/1015
59e02a29 (origin/pr/1015) changes to allow for darwin compilation
$ git grep 'func IsRootless' 59e02a29
59e02a29:libpod/container_darwin.go:func IsRootless() bool {
59e02a29:libpod/container_linux.go:func IsRootless() bool {
59e02a29:libpod/container_windows.go:func IsRootless() bool {
59e02a29:pkg/rootless/rootless.go:func IsRootless() bool {
```